stdlib/asm/crypto/stark/deep_queries.masm

```diff
@@ -282,7 +281,7 @@ proc.compute_q_z_p_gz_at_alpha
     ## 3) Load and process the last 2 extension field elements. 
     ##    We use `mem_stream` but we will discard the last 4 field elements.
     mem_stream
-    exec.constants::zero_word mem_loadw
+    exec.constants::zero_word_ptr mem_loadw
```

Comment:
Probably not for this PR, but I would potentially convert this into a single call as well - e.g., something like:
```
exec.constants::zeroize_stack_word
```

Reply:
Added the suggested procedure